devicetypes/smartthings/bose-soundtouch.src/bose-soundtouch.groovy

```diff
@@ -0,0 +1,989 @@
+/**
+ *  Bose SoundTouch
+ *
+ *  Copyright 2015 Henric Andersson
```

Comment:
Do we individually copyright SmartApps?  I had thought copyright for SmartApps written by employees would be assigned to "SmartThings" or "SmartThings, Inc.".